src/ir/block.h

```diff
@@ -0,0 +1,64 @@
+//-----------------------------------------------------------------------------
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//----------------------------------------------------------------------------
+#ifndef SRC_IR_BLOCK_H_
+#define SRC_IR_BLOCK_H_
+
+#include <vector>
+
+#include "absl/container/flat_hash_map.h"
+#include "src/ir/operation.h"
+#include "src/ir/operator.h"
+#include "src/ir/types/type.h"
+#include "src/ir/value.h"
+
+namespace raksha::ir {
+
+// A collection of operations.
+class Block {
+ public:
+  // The type for a collection of `Operation` instances.
+  using OperationList = std::vector<std::unique_ptr<Operation>>;
+
+  Block() = default;
```

Comment:
Why do we need this explicitly? If this line were gone, wouldn't it just fill in a default no-arg ctor?

Reply:
Ah, ok. Did not know that. Will resolve this and similar comments.